po/common/en.po

```diff
@@ -4685,7 +4685,7 @@ msgid "Processed foods"
 msgstr "Processed foods"
 
 msgctxt "attribute_nova_4_description_short"
-msgid "Ultra processed foods"
+msgid "Ultra-processed foods"
```

Comment:
changing the msgid does not work if it's not changed in common.pt too (in which case we lose the translations though..)
I replaced it in common.pot and all .po files